api/src/main/java/org/openmrs/module/fhir2/api/FhirServiceRequestService.java

```diff
@@ -14,13 +14,14 @@
 import ca.uhn.fhir.model.api.Include;
 import ca.uhn.fhir.rest.api.server.IBundleProvider;
 import ca.uhn.fhir.rest.param.DateRangeParam;
+import ca.uhn.fhir.rest.param.HasParam;
 import ca.uhn.fhir.rest.param.ReferenceAndListParam;
 import ca.uhn.fhir.rest.param.TokenAndListParam;
 import org.hl7.fhir.r4.model.ServiceRequest;
 
 public interface FhirServiceRequestService extends FhirService<ServiceRequest> {
 	
-	IBundleProvider searchForServiceRequests(ReferenceAndListParam patientReference, TokenAndListParam code,
+	IBundleProvider searchForServiceRequests(ReferenceAndListParam patientReference, TokenAndListParam code, HasParam has,
```

Comment:
nit: I'd prefer that the `has` param is near the end around where `includes` are (they are somewhat similar ideas.